src/Semantic/Api/Symbols.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE GADTs, TypeOperators, DerivingStrategies #-}
+{-# LANGUAGE MonoLocalBinds, RankNTypes #-}
```

Comment:
Out of curiosity, why `MonoLocalBinds` here?

Reply:
Single-instance typeclasses give you warnings without it.